addons/mail/static/tests/message/message_tests.js

```diff
@@ -672,6 +673,44 @@ QUnit.test("should not be able to reply to temporary/transient messages", async
     await contains(".o-mail-Message [title='Reply']", { count: 0 });
 });
 
+QUnit.test("squashed transient message should not have date in the sidebar", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "Channel 1" });
+    pyEnv["mail.message"].create([
+        {
+            body: "Hello world 1",
+            model: "discuss.channel",
+            res_id: channelId,
+            date: "2024-02-26 10:00:00",
+        },
+        {
+            body: "Hello world 2",
+            model: "discuss.channel",
+            res_id: channelId,
+            date: "2024-02-26 10:00:00",
+        },
+    ]);
+    const { openDiscuss } = await start();
+    await openDiscuss(channelId);
+    await click(".o-mail-Message.o-squashed");
+    await contains(".o-mail-Message.o-squashed .o-mail-Message-sidebar .o-mail-Message-date", {
+        text: deserializeDateTime("2024-02-26 10:00:00").toLocaleString(DateTime.TIME_24_SIMPLE),
+    });
```

Comment:
- `.o-mail-Message-date` looks useless, selector on `.o-squashed .o-mail-Message-sidebar` textContent is good enough
- isn't `10:00` enough? If you need to take timezone into account, could be worth `mockTimeZone(0)` as preamble of test so that the assertion is much easier to read with a simple `10:00` text content